docker-compose.yml

```diff
@@ -86,12 +86,13 @@ services:
     command:
```

Comment:
Let's move that magic into the file
```suggestion
    command:
      - -c
      - ./superscript.sh 
```

Reply:
Not sure if this is worth it now, with also the PR for initializing the operator set after deployment, I'm making a deployment script for this in another PR